components/build/tekton-results/.tekton/pvc.yaml

```diff
@@ -0,0 +1,11 @@
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: app-studio-default-workspace
```

Comment:
Does this need `namespace: build-service`?

Reply:
we are usually running the CI in the same namespace as the application is deployed.